WordPress/src/main/res/values-en-rAU/strings.xml

```diff
@@ -702,7 +702,7 @@ Language: en_AU
     <string name="activity_log_rewind_started_snackbar_message">Your site is being restored\nRewinding to %1$s %2$s</string>
     <string name="activity_log_button">Activity Log action button</string>
     <string name="plugin_auto_managed">Auto-managed</string>
-    <string name="reader_save_posts_locally_dialog_message">Save this post, and come back to read it whenever you\'d like. It will only be available on this device — saved posts don\'t sync to your other devices. (Yet! We\'re working on it.)</string>
+    <string name="reader_save_posts_locally_dialog_message">Save this post, and come back to read it whenever you\'d like. It will only be available on this device — saved posts don\'t sync to your other devices.</string>
```

Comment:
```suggestion
    <string name="reader_save_posts_locally_dialog_message">Save this post, and come back to read it whenever you\'d like. It will only be available on this device — saved posts don\'t sync to your other devices. (Yet! We\'re working on it.)</string>
```

We can revert this change and the release script will apply the proper changes.